platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,752 @@
+---
+title: "RNA-Seq"
+description: "An introduction to running nf-core/rnaseq in Seqera Platform"
+date: "21 Jul 2024"
+tags: [platform, compute environment, rnaseq, aws, gcp, azure]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide details how to run bulk RNA sequencing (RNA-Seq) data analysis, from quality control to differential expression analysis, on an AWS Batch compute environment in Platform. It includes: 
+
+- Creating an AWS Batch compute environment to run your pipeline and analysis data studio
+- Adding pipelines to your workspace 
+- Importing your pipeline input data
+- Launching the pipeline and monitoring execution from your workspace
+- Setting up a custom analysis environment with Data Studios
+- Resource allocation guidance for RNA-Seq data
+
+:::info[**Prerequisites**]
+You will need the following to get started:
+
+- [Admin](../orgs-and-teams/roles.mdx) permissions in an existing organization workspace. See [Set up your workspace](./workspace-setup.mdx) to create an organization and workspace from scratch.
+- An existing AWS cloud account with access to the AWS Batch service.
+- Existing access credentials with permissions to create and manage resources in your AWS account. See [IAM](../compute-envs/aws-batch.mdx#iam) for guidance to set up IAM permissions for Platform. 
+:::
+
+## Compute environment
+
+Compute and storage requirements for RNA-Seq analysis are dependent on the number of samples and the sequencing depth of your input data. See [RNA-Seq data and requirements](#rna-seq-data-and-requirements) for details on RNA-Seq datasets and the CPU and memory requirements for important steps of RNA-Seq pipelines. 
+
+In this guide, we will create an AWS Batch compute environment with sufficient resources allocated to run the [nf-core/rnaseq](https://github.com/nf-core/rnaseq) pipeline with a large (123.5 GB) dataset. This compute environment will also be used to run a Data Studios RStudio environment for tertiary analysis of the resulting pipeline data. 
```

Comment:
```suggestion
In this guide, you will create an AWS Batch compute environment with sufficient resources allocated to run the [nf-core/rnaseq](https://github.com/nf-core/rnaseq) pipeline with a large (123.5 GB) dataset. This compute environment will also be used to run a Data Studios RStudio environment for tertiary analysis of the resulting pipeline data. 
```
Changing this to you as we isn't used often and it feels weird to suddenly switch. Ignore if you disagree.